### ISSUE TITLE
Suggestion for use of area above warzone 

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#8]
Suggestion for use of area above warzone 

Implement a space travel mod idea : Put Venus and a couple of dwarf planets there. Accessible when at 600K XP or such.

**Comment by BuckarooBanzay** [comment]
What about some kind of magical place/planet with air and a fancy mapgen? Something like ethereal but on steroids :rainbow: :unicorn: 

Also: teleporting can be limited on an area basis, why not in a place like this, this encourages exploration and adds another difficulty level

**Comment by OgelGames** [comment]
> What about some kind of magical place/planet with air and a fancy mapgen? Something like ethereal but on steroids 🌈 🦄

I always thought it would be cool to have some ufos or something up there, but a fantasy world sounds even better 👀 Also I think there was some talk about dragons or something, could be a good place for them...

**Comment by Klaranth** [comment]
sunks — 03/22/2021
more planets, stars? adding them in worldgen maybe?
people would go crazy for them

wsor — 03/23/2021
aka more lua mapgen aka more lag
since i dont think you can abuse biomes to create "planets" like you can abuse them for caves

sunks — 03/23/2021
then some oxy in thin layer in space? priv to make spheres of uber cheap mat?
altho that'd backfire
can't break much with oxy thin layer, who would steal oxy anyway no such ppl
I may be on naive side and expect best of ppl

BuckarooBanzai — 03/23/2021
if you have an idea for a public star-system, feel free to propose it (available planet types are here: https://github.com/pandorabox-io/planetoidgen/#planet-types)
GitHub
pandorabox-io/planetoidgen
Contribute to pandorabox-io/planetoidgen development by creating an account on GitHub.
And here are the current constellations: https://github.com/pandorabox-io/pandorabox_custom/blob/master/planets.lua
GitHub
pandorabox-io/pandorabox_custom
pandorabox server customization mod. Contribute to pandorabox-io/pandorabox_custom development by creating an account on GitHub.

**Comment by Klaranth** [comment]
@S-S-X  suggested to do some tweaking and place Morlendor above the Warzone. 
Well, not planning to start working on it. But idea is good and if enough reusable stuff that can be used without complete rewrite is available then it might come real some day.

**Comment by Klaranth** [comment]
@OgelGames 
However, I would definitely like to see Morlendor added as a separate place one could go to fight dangerous mobs

**Comment by Klaranth** [comment]
Condensed results from a heated discussion in-game 

Planet mod can be made easily, according to BB, but e.g. Venus should have a few new features that the other planets don't have... 
- Extreme weather 
- Extreme heat to burn players when not wearing protective suit 
	- Get space suit upgraded.  (new recipes or issue #94 upgrades?) 
	- Add recipe for exoskeleton to use on new planet. 
- Geysers
- High Gravity / Pressure
- Heavy clouds 
- Acid atmosphere
- Radiation
- Molten stone
- Space junk bombardments
	- Every habitat on the surface needs a shield.
	- More frequent near the center, where a lot of players tend to gravitate to.
- By night it functions like the warzone, by day it doesn't.
- There is an open issue somewhere for pre-generated (loot?) ships on mapgen. 
- Add a unique mob 
- Dangers underground	
	- Molten ore
	- Lava titans ( From NSSM)
- Adding alien civilization - https://github.com/more-rubenwardy/alien 
	- Alien goo as an ore, and alien temples underground
- Teleport disfunction in random places. 
- Add different gases 
	- Add pollution and toxic stuff
		https://forum.minetest.net/viewtopic.php?t=13506

		
XP CAP - Suggested for 350.000 XP

Mapserver coverage (y-range?) planned for new planet:
New planet can use 18000 - 24000 in Y-range  
Which leaves still space for one extra addition ( Morlendor perhaps )


Alternative : Add Loria.
https://content.minetest.net/packages/siegment/loria/

**Comment by BuckarooBanzay** [comment]
> Alternative : Add Loria.
https://content.minetest.net/packages/siegment/loria/

Uh, this sadly doesn't work, thats a whole standalone game

**Comment by Klaranth** [comment]
> > Alternative : Add Loria.
> > https://content.minetest.net/packages/siegment/loria/
> 
> Uh, this sadly doesn't work, thats a whole standalone game

Darn.  
Cause that looked really nice.

**Comment by Klaranth** [comment]
FantasyWorld+Dragons - 5
 Extremely dangerous planets - 3
 Morlendor - 2
 Venus+dwarf planets - 2

 <VoteBot@bot> Votes for "Implementing above warzone":
 OgelGames — FantasyWorld+Dragons
 SX — Extremely dangerous planets
 Klaranth — Morlendor
 sivarajan — Venus+dwarf planets
 xman — Extremely dangerous planets
 PhrogMan — FantasyWorld+Dragons
 SolDeNoche — Venus+dwarf planets
 SamWorld — FantasyWorld+Dragons
 Lejo — Extremely dangerous planets
 SimonSays — FantasyWorld+Dragons
 Vamp — Morlendor
 marghl — FantasyWorld+Dragons

**Comment by OgelGames** [comment]
From `#offtopic`, thought it would be good to put here:

>1hit — 2022/01/06
>The server Illuna-TechEth has a "candy" biome, and you can mine "bacon ore" (makes really powerful tools).  That's kinda neat.
>I would not want ethereal on the normal earth for pbox.  And yes, its common and a bit overused.

**Comment by BuckarooBanzay** [comment]
I have currently no motivation to write an entire biome/layer from scratch, would be nice if there were some existing solutions that we can ~~steal~~borrow from

Anyway, about the candy/bacon/fantasy realm: would be a nice touch :+1:

**Comment by Klaranth** [comment]
birdlover32767
Also on the "things above warzone" stuff, maybe there could be 1000x1000x1000 random "planets"
That would make for 3844 of them, if they cover a plane, of course divide 17km-18km into 1km**2 squares and generate a flat planet inside each 

et 
Idea: naturally generating planetoids

**Comment by Klaranth** [comment]
Jeremy 
If someone wanted to get themselves into a lot of coding, you could implement something like a new planet with a new ore, which would only be refine-able or usable with a new type of voltage/technic machine, which could open up possibility to other new things, like new armors, new machines, new beacons, etc
Perhaps you could make a progression system so that the new machines are only obtainable using resources from the new planet.. But once you make them they can also help you with things from earth (normal ores) 

Marsufoman
Should an XP limit be well put there to access the planet?

Jeremy 
Well, we have been discussing what to put in warzone for a while.. :p Just don't know if anyone has the time/energy to code something of that sort. (above warzone*)

Marsufoman
I can only make textures etc. as im unexperienced with coding for the most part 

Jeremy 
Not to mention, making a new Tech Tree would also open up possibility to start using some resources that currently are not much in use; (Mithril, chernobolyte, diamond, sulfur, etc). i.e. Lots of room to expand.

Marsufoman
a more valuable dimension with more ores/recourses would be cool, as most of the time people just mine on asteroids or moon That well, gives more variety in what to do

**Comment by Klaranth** [comment]
A suggestion to use as a Fantasy World above Warzone (with an XP limit between 150K and 250K), or perhaps as inhabitants of Mars caves (no extra XP limit needed): 

https://content.minetest.net/packages/ElCeejo/draconis/

Adds massive, terrifying Dragons and powerful endgame equipment.

Dragons are massive flying beasts with powerful breath, capable of laying waste to almost anything.

Dragon Eggs are a rare and valuable drop from older Dragons. When put on a 3x3 nest of Bone Piles, the Egg will "awaken" and allow you to set it alight or drown it to begin hatching.

Draconic Steel can be crafted in to tools and armor (if stu's 3d armor mod is enabled) that far exceed the capabilities of diamond. Draconic Steel is costly, time-consuming, and unstackable. Each ingot is directly tied to the Dragon that forged it, and ingots forged by different individual Dragons cannot be crafted together. Draconic Steel tools gain power from their linked Dragon, with attack strength and speed increasing the closer the tool is. Inversely, if the Dragon dies the tool will shatter upon use.

More in-depth information can be found in the Libri Draconis item.

**Comment by BuckarooBanzay** [comment]
> A suggestion to use as a Fantasy World above Warzone (with an XP limit between 150K and 250K), or perhaps as inhabitants of Mars caves (no extra XP limit needed):
>
https://content.minetest.net/packages/ElCeejo/draconis/

i haven't checked it, but it would be nice to know if the spawning of the dragons can be limited to a y-range (for example from y=18k to y=20 to spawn in a new y-layer) :thinking:

**Comment by Klaranth** [comment]
Marsufoman 
Like many suggestions round here this wasn't actually related to other discussions, and not sure if it's serious suggestion really. Unique items would likely be hard idea in minetest setting.
Also no idea what it would do or how it would work.

**Comment by Klaranth** [comment]
> > A suggestion to use as a Fantasy World above Warzone (with an XP limit between 150K and 250K), or perhaps as inhabitants of Mars caves (no extra XP limit needed):
> 
> https://content.minetest.net/packages/ElCeejo/draconis/
> 
> i haven't checked it, but it would be nice to know if the spawning of the dragons can be limited to a y-range (for example from y=18k to y=20 to spawn in a new y-layer) 🤔

According to 1hit, there is not a config option for Draconis to limit the spawn x/y/z coords, just the spawn rate.

**Comment by Klaranth** [comment]
Added new mobs: https://codeberg.org/tenplus1/dmobs

**Comment by TheEt1234** [comment]
what about it being
- the opposite of warzone
- ethereal-like floatlands
- with dungeons
- maybe an unique resource
  - my first idea was ~~mese mk2~~ diginium
  - it would be used to craft some digiline devices idk, maybe i could make digibuilder mk2 that has it's delay based on max lag or something like that
- naturally generating planetoids
- Maybe the dungeon reward could be xp ;)
   - if this happens, maybe lower the entry to something like 100k xp, same as warzone
   
I could volunteer to develop this but i have very little experience with modding

**Comment by TheEt1234** [comment]
@Klaranth what do you think of this idea

**Comment by S-S-X** [comment]
@TheEt1234 idea is good sure, and has some similarities with some other suggestions.

But I'd like to point out few things:

> * the opposite of warzone
> * ethereal-like floatlands
> * with dungeons

All good so far, shouldn't be too complicated and all of this would be at the very core of the area.

> * maybe an unique resource

Maybe and would be nice but this probably should not be added with new area unless you happen to have ready made suggestion that have been proven to work and are actually useful for players. Simple new nodes for building could work but anything besides that would likely be too complicated and should be developed separately and possibly added later.

>   * my first idea was ~mese mk2~ diginium
>   * it would be used to craft some digiline devices idk, maybe i could make digibuilder mk2 that has it's delay based on max lag or something like that

This is what I mean with too complicated: introducing new stuff for crafting and other significant changes unrelated to actual area requires a lot of work to get right. I'd suggest forget about these here but possibly open separate idea ticket for these crafting/upgrade/etc. things.

> * naturally generating planetoids

Probably fine if you could use ready made or semi ready code (there's been planetoids on pandorabox, might want to take a look at those to see if it would fit here).

> * Maybe the dungeon reward could be xp ;)

Not exactly related to new area and should be developed separately -> another idea ticket.
There's few other XP related tickets you might want to take a look at.

>   * if this happens, maybe lower the entry to something like 100k xp, same as warzone

This one is easy and can be determined afterwards, there's ready made templates for simple travel limits.

> I could volunteer to develop this but i have very little experience with modding

👍 when I started I also had very little experience 😄

**Comment by Klaranth** [comment]
> @Klaranth what do you think of this idea

Not up to me. You got some in-depth reply from @S-S-X. I just copy/paste #ideas in here, so the coders can browse for their next project. :-D

**Comment by Klaranth** [comment]
Hedgehog 
Interesting : dmobs can give you the opportunity to tame dragons : https://codeberg.org/tenplus1/dmobs

**Comment by TheEt1234** [comment]
i have forgotten about this uh... sorry for the delay


so adding "ethereal floatlands" is sort of simple
i edited ethereal to basically limit its biomes to the floatland area, and limit the 
default biomes to be under it 
also enabled floatland gen aand it magically worked
(is using floatlands fine for this?)

i don't know if this messes with mars or planetoids

There's also the question of which biomes do we enable, as there are some boring ones and some really cool ones

also, ethereal adds 2 new resources:
- etherium (whose sole pourpourse is to make flight potions [ i don't know how beacons mess with flight potions, but they can be disabled ], also the recipe could probably just be remade to just use crystal ingots ~~also would be funny if the server switched to using that as currency~~)
- crystal, it's pourpourse is:
  - armor (yes THAT crystal armor, also boots give you speed bost) and tools
  - crystal gilly staff
 And crystal ingots are made from combining 2 mese ores, 2 crystal spikes [ decaying crystal grass ] and one water bucket
 
i can publish my current very scuffed fork of ethereal to github if anyone wants to check it out


After some playing around i also noticed that fly potions would actually be useful, as they are stackable and by default last 300 seconds

**Comment by SwissalpS** [comment]
I don't see the flight potions fitting into pandorabox.
Similar with the crystal armour and tools. There are already so many on pandorabox.

Would having two types of speed-shoes actually add or dilute? The crystal ones would only be obtainable by high XP players or through merchandising.

Wha'ts the 'crystal gilly staff' for?

Yes do post your fork :D

**Comment by TheEt1234** [comment]
> Wha'ts the 'crystal gilly staff' for?
> Yes do post your fork :D

Will do all of that tomorrow

**Comment by TheEt1234** [comment]
so apparently it replenishes oxygen when used, i think it could have a more obvious name like "staff of breathing"

but i disagree with crystal ingots being removed as i can add more uses for them in the future

**Comment by TheEt1234** [comment]
i also published my fork to https://github.com/TheEt1234/ethereal-pandorabox [also, i did `git init`..... maaaybe that was a huge mistake.... weee will see later]

**Comment by SwissalpS** [comment]
> i did `git init`..... maaaybe that was a huge mistake

yeah, that is sub-obtimal. It makes it a bit harder to see what you actually changed.

> but i disagree with crystal ingots being removed as i can add more uses for them in the future

I don't have anything against the crystals or the ingots. It's the armour, potion and the staff that I'm unsure actually add anything to pandorabox. (I may be wrong, just my two cents)

**Comment by SwissalpS** [comment]
for example, there are 5 different hoes on pandorabox. I only ever use the iron version. I've tried the others, but none of them gives me any reason to use them. The wooden one is great until you get stone but by then you basically already have iron.

That's the kind of thing that makes me wary to add new recipes for things that already exist unless they solve a niche that was anoyingly missing.

- Breath beacons and spacesuits work fine, imo no need for yet another breathing device.
- Flying is well covered with beacons gliders and xenos.
- Armour is already hugely overpopulated to the point that players opt for one or the other for fashion not function. (me included)

**Comment by TheEt1234** [comment]
i see, well i think we should think about the uses for crystal ingots
> yeah, that is sub-obtimal. It makes it a bit harder to see what you actually changed.

~~i will try to somehow solve this somehow~~ solved by copying the ".git" directory from original ethereal, then making a commit

#### Also, keep in mind, ethereal requires some configuration to be usable, and especially this version [like disabling swamps or coral reefs and ofc disabling overriding default biomes]

**Comment by TheEt1234** [comment]
So, i tested this with a pandorabox-like setup (pandorabox-io/pandorabox-mods, limited ethereal and floatlands to 18 000 - 20 000) and amazingly it didn't *seem* to break anything and worked as intended, but i think it might break something above it if it uses default biomes

#### what should i do next
currently i have ethereal inside floatlands mapgen, in the y coordinates between 18 000 - 20 0000

**Comment by BuckarooBanzay** [comment]
> what should i do next

can you give me your settings so i could set the test-server up with your fork (this isn't a "yes-we-are-doing-this" it's just to see how it looks like:)

**Comment by Klaranth** [comment]
Hedgehog 
Interesting : dmobs can give you the opportunity to tame dragons : https://codeberg.org/tenplus1/dmobs

**Comment by TheEt1234** [comment]
> > what should i do next
> 
> can you give me your settings so i could set the test-server up with your fork (this isn't a "yes-we-are-doing-this" it's just to see how it looks like:)

https://github.com/TheEt1234/ethereal-pandorabox/blob/master/map_meta_i_used.txt
in minetest.conf: 
```
ethereal.clear_default_biomes = false
ethereal.leaftype = 1
ethereal.xcraft = false
ethereal.leafwalk = true
ethereal.quicksand = 0
ethereal.jumble = 0
ethereal.sakura = 0
ethereal.reefs = 0
ethereal.sealife = 0
ethereal.glacier = 0
ethereal.sandstone = 1
ethereal.snowy = 0
ethereal.desert = 0
ethereal.cavedirt = false
ethereal.mediterranean = 0
ethereal.swamp = 0
ethereal.junglee = 0
ethereal.grassytwo = 0
ethereal.papyruswalk = false
```

**Comment by Klaranth** [comment]
birdlover32767
so i think the ethereal floatlands will be made from a 3d perlin noise, with biomes and maybe some new stuff

Hedgehog 
The ethereal floatlands, is that an idea or a suggestion on how to build it ?

birdlover32767
I guess how to build it
Messing around with mapgen rn to learn how it works

et 
The ethereal floatlands are currently using... well... floatlands. And that works good enough️.

birdlover32767
about that floatlands etheral suggestion, maybe we can use etherium as the new resource
in ethereal it's only used for the flight potion but maybe it can also be used for other magical stuff

et 
make etherium the server currency
make it so if you do heavy luacontroller processing, like reach a timeout or something, it will drop one etherium
i don't think my idea will lead to any harm of the server at all

**Comment by TheEt1234** [comment]
> et
> make etherium the server currency
> make it so if you do heavy luacontroller processing, like reach a timeout or something, it will drop one etherium
i don't think my idea will lead to any harm of the server at all

(also that one was a joke)

**Comment by TheEt1234** [comment]
> can you give me your settings so i could set the test-server up with your fork (this isn't a "yes-we-are-doing-this" it's just to see how it looks like:)

also, the entire map meta doesn't need to be copied, just stuff related to floatlands

**Comment by TheEt1234** [comment]
> I don't have anything against the crystals or the ingots. It's the armour, potion and the staff that I'm unsure actually add anything to pandorabox. (I may be wrong, just my two cents)

So it turns out ethereal tools ""preserve"" the node, so when you dig dirt with grass, you get dirt with grass in your inventory instead of just normal dirt

**Comment by SwissalpS** [comment]
> So it turns out ethereal tools ""preserve"" the node, so when you dig dirt with grass, you get dirt with grass in your inventory instead of just normal dirt

OK, cool, they have some unique reason to exist then.

So, what happens if you dig "digiline button on" or similar (piston extended, stone with gold ore)
Does it break anything to then place that on-state button or device?

**Comment by SwissalpS** [comment]
coral is another one that can't be preserved when digging.

**Comment by TheEt1234** [comment]
> > So it turns out ethereal tools ""preserve"" the node, so when you dig dirt with grass, you get dirt with grass in your inventory instead of just normal dirt
> 
> OK, cool, they have some unique reason to exist then.
> 
> So, what happens if you dig "digiline button on" or similar (piston extended, stone with gold ore)
> Does it break anything to then place that on-state button or device?

So turns out, it seems like it's only for the shovel (but the ability can be implemented somewhat easily with a pickaxe using a list)

But i observed the same behaviour with the multitool, i will need to test it (i'll get to it in about one hour)

https://github.com/TheEt1234/ethereal-pandorabox/blob/master/crystal.lua#L230

**Comment by TheEt1234** [comment]
So turns out that the shovel can preserve anything in the crumbly group (easily searchable in the unified inventory) and doesn't dig anything in the no_silktouch group

.... But the **ethereal multitool also digs cracky nodes....** and *doesn't care about the no_silktouch group (nothing is in it anyway)*
I haven't managed the crystal tool to dig digilines buttons *(probably because of my bad testing)* but i have **managed to get them to dig "Telemosaic Beacon Active (you hacker you!)"** 
so i guess the multitool *(and the other tools to give them a use)* will need to use a list of nodes or maybe something scuffed like checking if the node name starts with "default:" 
*so what aproach should i take to fix this*

And there's also https://github.com/ChimneySwift/multitools/pull/8

> coral is another one that can't be preserved when digging.

Why?

**Comment by SwissalpS** [comment]
normally coral loses its colour when dug.

Edit: iirc it turns into coral-skeleton or similar

**Comment by SwissalpS** [comment]
> to fix this

easiest is to not add the tool :p
 
otherwise the tool may need to be made to respect an allow-only-these-nodes list, or a deny-digging-these-nodes/groups list.
Both approaches aren't that easy to maintain though.

I'm not sure it would be sufficient to check if the node is in the 'in creative inventory' group.

**Comment by TheEt1234** [comment]
> normally coral loses its colour when dug.
> 
> Edit: iirc it turns into coral-skeleton or similar

why would preserving the coral be an issue, it could be used for decoration

**Comment by TheEt1234** [comment]
> > to fix this
> 
> easiest is to not add the tool :p
> 
> otherwise the tool may need to be made to respect an allow-only-these-nodes list, or a deny-digging-these-nodes/groups list. Both approaches aren't that easy to maintain though.
> 
> I'm not sure it would be sufficient to check if the node is in the 'in creative inventory' group.

An allow list and the shovel's crumbly checking would probably be fine, i mean with a deny list you actively have to maintain it or something will break

**Comment by SwissalpS** [comment]
> why would preserving the coral be an issue, it could be used for decoration

The developers put thought and effort into that node to be more realistic. Coral is delicate, it doesn't like to be moved and certainly isn't going to keep its colour when placed in the sun/air :D

Being able to dig dirt_with_grass and co, is more realistic.

We used to be able to dig piston heads, vacuum, ... but those have been ristricted.
If we add a tool that allows players to get those again, it would only be a matter of time that the tool gets removed or restricted. So we might as well check these issues first.

**Comment by TheEt1234** [comment]
> The developers put thought and effort into that node to be more realistic. Coral is delicate, it doesn't like to be moved and certainly isn't going to keep its colour when placed in the sun/air :D

Argh realism.... but fair ~~though the crystal spike inside the crystal ingots has magik properties that keep the coral alive :)~~ 

> If we add a tool that allows players to get those again, it would only be a matter of time that the tool gets removed or restricted. So we might as well check these issues first.

Yes, i am aware of that, i should patch these issues

> We used to be able to dig piston heads, vacuum, ... but those have been restricted.

Also fun fact: You are actually still able to dig digiline piston heads i think

**Comment by TheEt1234** [comment]
> So turns out that the shovel can preserve anything in the crumbly group (easily searchable in the unified inventory) and doesn't dig anything in the no_silktouch group
> 
> .... But the **ethereal multitool also digs cracky nodes....** and _doesn't care about the no_silktouch group (nothing is in it anyway)_ I haven't managed the crystal tool to dig digilines buttons _(probably because of my bad testing)_ but i have **managed to get them to dig "Telemosaic Beacon Active (you hacker you!)"** so i guess the multitool _(and the other tools to give them a use)_ will need to use a list of nodes or maybe something scuffed like checking if the node name starts with "default:" _so what aproach should i take to fix this_
> 
> And there's also [ChimneySwift/multitools#8](https://github.com/ChimneySwift/multitools/pull/8)
> 
> > coral is another one that can't be preserved when digging.
> 
> Why?

So, i've made it so that *all* ethereal tools get this silk touch ability
It checks for all of these before silktouching:
the node that it's digging:
   - isn't in not_in_creative_inventory *already filters out pretty much everything bad*
  - isn't in no_silktouch group (unused)
  - is in the soil group or starts with `"default:"` `"ethereal:"` `"moreores:"` `"technic:mineral"`
  
  I will try to figure out how to override the multitool's problematic behaviour and then push it to github

**Comment by TheEt1234** [comment]
So, the only way i was able to solve this was creating my own multitools fork https://github.com/TheEt1234/multitools that removes the problematic crystal multitool

The multitool is registered at my ethereal fork now

**Comment by TheEt1234** [comment]
@BuckarooBanzay if you have the time, please read the messages above, and configure the test server as i've described

TL;DR
- My fork of ethereal is optionally dependant on https://github.com/TheEt1234/multitools (because the original multitool's behaviour had been problematic and allowed you to silk touch nodes like active telemosaics) 
    - My multitools fork removes the registration of the crystal multitool and dependance on ethereal, this is done on the ethereal side now
- The map_meta.txt needs to have theese lines for the floatlands to generate where they need to be:
```
mgv7_floatland_ymin = 18000
mgv7_floatland_ywater = -31000
mgv7_floatland_taper = 256
mgv7_floatland_ymax = 20000
mgv7_floatland_density = -0.300000012
```
and floatlands need to be enabled (i haven't found the option for that)

- Optional (but recomended) settings to minetest.conf
(the clear_default_biomes=false is required)
```
ethereal.clear_default_biomes = false
ethereal.leaftype = 1
ethereal.xcraft = false
ethereal.leafwalk = true
ethereal.quicksand = 0
ethereal.jumble = 0
ethereal.sakura = 0
ethereal.reefs = 0
ethereal.sealife = 0
ethereal.glacier = 0
ethereal.sandstone = 1
ethereal.snowy = 0
ethereal.desert = 0
ethereal.mediterranean = 0
ethereal.swamp = 0
ethereal.junglee = 0
ethereal.grassytwo = 0
ethereal.papyruswalk = false
```
- Remove "The area above 18000 is reserved for future use, sorry..." when you try to go above y=18000

**Comment by birdlover32767** [comment]
ethereal is nice and all but there needs to be a reason to go there other than to flex on others that haven't yet

maybe a new resource _(etherium)_ for some exotic machines/equipment or some structures with loot could be good

**Comment by TheEt1234** [comment]
> ethereal is nice and all but there needs to be a reason to go there other than to flex on others that haven't yet

absolutely, but what exactly should we add

**Comment by BuckarooBanzay** [comment]
> @BuckarooBanzay if you have the time, please read the messages above, and configure the test server as i've described

yeah, sorry, i don't have too much time on my hands right now, need to prioritize things i think :/

if you want to test this locally please use the public map-backup and change the (also public) mod-collection accordingly

**Comment by TheEt1234** [comment]
So i finally found a point in the custom mapgen, it can look amazing 
![image](https://github.com/pandorabox-io/in-game/assets/74461678/09a0e6e9-7bcb-425a-9677-c43d7ba5ade5)

(the picture shows a modified version of the example lua mapgen {in contentdb}, where i changed one funny value from `128` to `5 000`)

**Comment by AnarchyEnthusiast** [comment]
Semi-finalized idea for content above warzone: 
https://github.com/pandorabox-io/in-game/issues/398
